src/registrar/models/domain_information.py

```diff
@@ -75,6 +75,24 @@ class Meta:
         verbose_name="Suborganization",
     )
 
+    requested_suborganization = models.CharField(
```

Comment:
**blocking** please remove this from domain info, we don't need to store it here.

Reply:
this is per [this](https://cisa-corp.slack.com/archives/C05BGB4L5NF/p1730392283781909) slack thread